src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -155,9 +158,8 @@ typedef struct wrapped_rr_closure_arg {
   /* The RR instance related to the closure */
   grpc_lb_policy *rr_policy;
```

Comment:
I just noticed that this field is probably not needed.  We're setting it in a bunch of places, but it doesn't look like we're actually using it anywhere.  (Well, we are using it in a couple of places where we should be using `glb_policy->rr_policy` instead, but that's easily fixed.)